misc/python/materialize/test_analytics/connection/test_analytics_connection.py

```diff
@@ -45,8 +45,15 @@ def execute_updates(
     sql_statements: list[str], cursor: Cursor, verbose: bool = False
 ) -> None:
     for sql in sql_statements:
-        sql = dedent(sql)
-        if verbose:
-            print(f"> {sql}")
-
-        cursor.execute(sql)
+        try:
+            sql = dedent(sql)
+            if verbose:
+                print(f"> {sql}")
```

Comment:
We might want to always print this in the beginning so that we have a written record of what we INSERTed into the db etc.

Reply:
Sure